### PR TITLE
Locale overhaul

### PR DESCRIPTION
**Pull request by forgetso**
- [x] separate locale from common package
- [x] separate locale into [JSON + backend] package and frontend package
- [x] add test to check if locale JSON strings are up to date

### CODE REVIEW

File: packages/database/src/databases/provider.ts [hunk 1]
```diff
@@ -16,8 +16,10 @@ import {
 	type Logger,
 	ProsopoDBError,
 	ProsopoEnvError,
-	type TranslationKey,
 } from "@prosopo/common";
+importtype {
```
Comment (goastler) [inline packages/database/src/databases/provider.ts]: typo

File: packages/locale-browser/.npmignore
Comment (goastler) [inline packages/locale-browser/.npmignore:1]: delete pls

File: packages/locale-browser/vite.test.config.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import fs from "node:fs";
```
Comment (goastler) [inline packages/locale-browser/vite.test.config.ts]: imports above license

File: packages/locale/.npmignore
Comment (goastler) [inline packages/locale/.npmignore:1]: remove pls

**Review by goastler — CHANGES_REQUESTED** [review]

File: packages/database/src/databases/provider.ts (hunk 1 above)
Comment (forgetso) [reply packages/database/src/databases/provider.ts]: Resolved. It was breaking the build.

File: packages/locale/.npmignore
Comment (forgetso) [reply packages/locale/.npmignore:1]: Removed

File: packages/locale-browser/vite.test.config.ts (hunk 2 above)
Comment (forgetso) [reply packages/locale-browser/vite.test.config.ts]: Sorted

File: packages/locale-browser/.npmignore
Comment (forgetso) [reply packages/locale-browser/.npmignore:1]: Removed

**Review by goastler — APPROVED** [review]

**Pull request merged**